recipe/conda_build_config.yaml

```diff
@@ -1,3 +1,4 @@
 mpi:
   - openmpi
   - mpich
+  - msmpi
```

Comment:
We'll probably want to add a `# [win]` selector, unless MS MPI also works on Linux and macOS.

Reply:
Thanks! `msmpi` is only on Windows and the other two are not. I added selectors for all of them.